### DISCUSSION TITLE
Dynamically creating models using the FieldInfo definition and constraints?

### DISCUSSION THREAD

**Discussion by stur86** [pydantic/pydantic#4030] [Question]
I'm trying to write code that creates models dynamically, as I need to assign certain properties to them on the fly. However if I run this simple script:

```python
from pydantic import create_model, Field

MyModel = create_model('MyModel', x=Field(0, ge=0))
```

I get an error because the `ge` constraint is not enforced. Is there any way to get this working instead?

**Comment by gryevns** [comment]
```python
from pydantic import create_model, Field

MyModel = create_model("MyModel", x=(int, Field(0, ge=0)))

MyModel(x=-1)  # validation error
```
https://pydantic-docs.helpmanual.io/usage/models/#dynamic-model-creation

**Reply by stur86** [reply]
Wait, so this works if I pass it in tuple form, but not just with the Field? Is it because it needs the type explicitly to apply the constraint?
